src/services/preProcess.ts

```diff
@@ -25,6 +26,7 @@ namespace ts {
             else if (token === SyntaxKind.CloseBraceToken) {
                 braceNesting--;
             }
+            lastTokenWasDot = token === SyntaxKind.DotToken;

```

Comment:
should not this be before we consume the next token? this just says `token === SyntaxKind.DotToken`